src/mocks/botframework/index.js

```diff
@@ -199,39 +202,38 @@ var appTest = express()
 appTest.use(bodyParser.json())
 
 appTest.get('/', (req, res) => {
-  var urlparts = url.parse(webhookurl)
+  var urlparts = url.parse(botHealthCheckUrl)
 
   tcpPortUsed.check(parseInt(urlparts.port), urlparts.hostname)
     .then((inUse) => {
-      console.log('port usage chatbot endpoint (' + webhookurl + '): ' + inUse)
+      console.log('port usage chatbot endpoint (' + botHealthCheckUrl + '): ' + inUse)
       if (inUse) {
-        console.log('checking chatbot endpoint (' + webhookurl + ') for response')
+        console.log('checking chatbot endpoint (' + botHealthCheckUrl + ') for response')
         var options = {
-          uri: webhookurl,
-          method: 'POST',
-          json: { },
+          uri: botHealthCheckUrl,
+          method: botHealthCheckVerb,
+          json: {},
           headers: {
             'Authorization': 'Bearer ' + securityToken
           }
         }
         request(options, (err, response, body) => {
           if (err) {
-            var offlineMsg = 'chatbot endpoint (' + webhookurl + ') not yet online (Err: ' + err + ', Body: ' + body + ')'
+            var offlineMsg = 'chatbot endpoint (' + botHealthCheckUrl + ') not yet online (Err: ' + err + ', Body: ' + body + ')'
             console.log(offlineMsg)
             res.status(500).send(offlineMsg)
-          } else {
-            var onlineMsg = 'chatbot endpoint (' + webhookurl + ') online (StatusCode: ' + response.statusCode + ', Body: ' + body + ')'
-            console.log(onlineMsg)
-            res.status(200).send(onlineMsg)
           }
+          var onlineMsg = 'chatbot endpoint (' + botHealthCheckUrl + ') online (StatusCode: ' + response.statusCode + ', Body: ' + body + ')'
```

Comment:
either add a "return" statement above when sending the status 500, or add the else-branch again. otherwise the status would be sent twice, resulting in an error. ("res.status(500)..." doesn't mean that the rest of the function won't be ran)